chunks/lang-node/Dockerfile

```
@@ -9,12 +9,15 @@ USER gitpod
 ENV TRIGGER_REBUILD=1
 ENV NODE_VERSION=${NODE_VERSION}
 
+ENV PNPM_HOME=/home/gitpod/.pnpm
+ENV PATH=/home/gitpod/.nvm/versions/node/v${NODE_VERSION}/bin:/home/gitpod/.yarn/bin:${PNPM_HOME}:$PATH
+
 RUN curl -fsSL https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | PROFILE=/dev/null bash \
     && bash -c ". .nvm/nvm.sh \
         && nvm install v${NODE_VERSION} \
         && nvm alias default v${NODE_VERSION} \
-        && npm install -g typescript yarn pnpm node-gyp" \
+        && npm install -g yarn \
+        && yarn global add typescript yarn pnpm node-gyp" \
```

I think we don't need a typescript here. But, yeah other packages should be installed globally.
```suggestion
        && yarn global add yarn pnpm node-gyp" \
```

---

`pnpm` can vary by node version too, https://pnpm.io/installation#compatibility, therefore, we should not install it with `yarn`, because then it won't change when we switch with `nvm`.

```suggestion
        && yarn global add yarn node-gyp" \
```

I would be fine with :point_up: , because `node-gyp` depends on the python version, and will not change with `nvm`.

---

https://github.com/nodejs/Release Most tools drop support for un-maintained versions.

---

If I move typescript install using npm. Is this PR can be approvoed ? @kylos101

---

@kylos101 ping

---

Friendly ping.

---

Hey @gengjiawen, I understand that you're excited to get merged and I appropriate that you are 🙏 

@kylos101 is on vacation but will be back in a few days!

> can you help on this one?

I would love to but I'm not sure how can I help further here, I do not maintain the `workspace-images`, team-workspace does 🙏

---

ping again.

---

@sagor999 Anything you can help here ?